tests/WorkerTest.php

```diff
@@ -44,13 +45,18 @@ public function testGetHandler()
 
     public function testGetHandlerNotCallable()
     {
+        $this->setExpectedExceptionRegExp(
+            HandlerException::class,
+            '/The handler for `test` is invalid./'
```

Comment:
Don't you want to use `expectException`? I thought `setExpectedException` was deprecated.